tracr/io/util.py

```diff
@@ -1,23 +1,35 @@
-import os
+"""
+File reading tools for decomposing input file name. Returns format string
+depending on extension (either '.tif/.tiff' or '.dcm/.dicom').
+
+INPUT:
+    - string containing file name (no folders make it here)
+
+OUTPUT:
+    - string declaring extension (tif or dcm)
+"""
 
+import os
 
 def guess_format(filename):
-    # guess the format of the file
+    # Guess the format of the file
     pathstr, basename = os.path.split(filename)
     name, ext = os.path.splitext(basename)
     ext = ext.lower()
     # list recognized filenames
     if ext in ('.tif', '.tiff'):
         return 'tif'
+    elif ext in ('.dcm', '.dicom')
+        return 'dcm'
     else:
         msg = 'Could not guess the format of {}.'.format(filename)
         raise IOError(msg)
 
```

Comment:
Is IOError appropriate here? Is it an error if the format guesser cannot guess the format, or should it simply return an empty string (keeping the return type constant)? What construction do we anticipate the calling function will use? E.g.

```
fmt = guess_format('foo.tif')
if fmt == 'tif':
    read_tif('foo.tif')
elif fmt == 'dcm':
    read_dcm('foo.dcm')
else:
    raise NotImplementedError(...)
```

or

```
reader = {
    'dcm' : read_dcm,
    'tif'     : read_tif }.get(guess_format(filename, None))
if reader is None:
    raise NotImplementedError(...)
```

I guess I just like these options better than wrapping the format guesser in`try-except` constructions.